packages/shared/src/contexts/AuthContext.tsx

```diff
@@ -101,9 +101,9 @@ const logout = async (reason: string): Promise<void> => {
 export type AuthContextProviderProps = {
   user?: LoggedUser | AnonymousUser;
   isFetched?: boolean;
-  isPastRegistration?: boolean;
```

Comment:
We also have `setIsRegistration` in `AuthOptions.onProfileSuccess`? https://github.com/dailydotdev/apps/commit/80281fe95eafaefa87846dae7bff3f7e12037291#diff-0d72ac8004175d51542cd27ed8342d244a8e8e7f26876ad22bca3cb57a62ec9fR247

Reply:
This is different; we use this to check whether the user should be logged out.